Makefile

```diff
@@ -13,3 +13,15 @@ test:
 
 vet:
 	go vet ./...
+
+# Docker commands
```

Comment:
Let's drop this comment — it doesn't really add any additional information given that the targets have a `docker*` prefix already.